EDI/Overview/03_TipsTricks.md

```diff
@@ -0,0 +1,74 @@
+# Tips and tricks for preparation
+
+
+
+## Introduction
+
+UN/EDIFACT (in the following short EDIFACT) is a cross-industry international standard for the format of electronic data in business transactions. EDIFACT is one of several international EDI standards. A UN organization called CEFACT, which is affiliated to the UNECE, is responsible for the EDIFACT standard.
+[^1]    
+
```

Comment:
Vorschlag: The United Nations Centre for Trade Facilitation and Electronic Business (UN/CEFACT) , subsidiary, intergovernmental body of the United Nations Economic Commission for Europe (UNECE), ... 
Quelle: https://unece.org/trade/uncefact
Ich fänd diese Seite als Quelle zu erwähnen "seriöser" als Wikipedia. Aber brauchen wir unbedingt hier Wikipedia/eine Quelle zu zitieren?
Die Fußnote und die "Callout" werden in GitHub nicht richtig angezeigt (also, kein Link).